pkg/controller/migplan/migplan_controller.go

```diff
@@ -39,7 +39,10 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
-const MigplanRunning = "migplan-running"
+const (
```

Comment:
can you define this consts in v1alpha1/model.go?

Use it there and then import it here as well if you need them

Reply:
Done